examples/preprocessing/plot_ica_from_epochs.py

```diff
@@ -8,6 +9,9 @@
 The sources matching the ECG are automatically found and displayed.
 Subsequently, artefact detection and rejection quality are assessed.
 Finally, the impact on the evoked ERF is visualized.
+
+Note that this example does quite a bit of processing, so even on a very
+fast machine it can take over a minute to complete.
 """
```

Comment:
The ICA fit takes a 34 seconds on my laptop (mkl + anaconda).

2015-05-14 19:18 GMT+02:00 Denis-Alexander Engemann <
denis.engemann@gmail.com>:

> last comment relates to SPM data examples
>
> 2015-05-14 19:18 GMT+02:00 Denis-Alexander Engemann <
> denis.engemann@gmail.com>:
>
>> Need to re-download data, telling you in a bit.
>>
>> 2015-05-14 19:16 GMT+02:00 Eric Larson <notifications@github.com>:
>>
>>> In examples/preprocessing/plot_ica_from_epochs.py
>>> <https://github.com/mne-tools/mne-python/pull/2092#discussion_r30340543>
>>> :
>>>
>>> > @@ -8,6 +9,9 @@
>>> >  The sources matching the ECG are automatically found and displayed.
>>> >  Subsequently, artefact detection and rejection quality are assessed.
>>> >  Finally, the impact on the evoked ERF is visualized.
>>> > +
>>> > +Note that this example does quite a bit of processing, so even on a very
>>> > +fast machine it can take over a minute to complete.
>>> >  """
>>>
>>> ...and it's not a slow computer. How long does it take for you?
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/mne-tools/mne-python/pull/2092/files#r30340543>.
>>>
>>
>>
>